src/daemon/ca/events.rs

```diff
@@ -427,9 +427,16 @@ impl AspaObjectsUpdates {
 /// Describes an update to the set of ROAs under a ResourceClass.
 #[derive(Clone, Debug, Default, Deserialize, Eq, PartialEq, Serialize)]
 pub struct ChildCertificateUpdates {
+    #[serde(skip_serializing_if = "Vec::is_empty", default)]
```

Comment:
These fields were not previously omitted from the serialization. Will the deserializing party handle this difference?

Reply:
yes, that would be here and 'default' ensures this.